### ISSUE TITLE
test accuracy

### ISSUE THREAD

**Issue by ZooHam** [V-Sense/ACTION-Net#9]
Hi, I have downloaded your jester-resnet pretrained model and jester annotations. I find the lables in test.pkl are all 0. When I run the test_jester_8f.sh, I get top1 =0, top5 = 100 witch mode in test.py is 'test' and use the jester_annotation test.pkl.
![image](https://user-images.githubusercontent.com/26896476/128451607-cc6be099-6990-4ab7-9060-fe537ac73379.png)

![image](https://user-images.githubusercontent.com/26896476/128451895-37c7458c-c86a-4419-9937-e0362d4103ba.png)

**Comment by villawang** [comment]
> Hi, I have downloaded your jester-resnet pretrained model and jester annotations. I find the lables in test.pkl are all 0. When I run the test_jester_8f.sh, I get top1 =0, top5 = 100 witch mode in test.py is 'test' and use the jester_annotation test.pkl.
> ![image](https://user-images.githubusercontent.com/26896476/128451607-cc6be099-6990-4ab7-9060-fe537ac73379.png)
> 
> ![image](https://user-images.githubusercontent.com/26896476/128451895-37c7458c-c86a-4419-9937-e0362d4103ba.png)

Hi there, you are correct. All test labels are 0 because the test data label is not visible. We normally report the accuracy on the validation set. So you can change the 'test' to 'val'. You should be able to get the correct results.

**Comment by ZooHam** [comment]
Thank you for your answer! Now, I want to reproduce your work. But I have only two 12G Graphics cards, and The maximum value of batch_size can only be set to 16.   Do you have any suggestion about hyperparameters setting during model training in order to get the accuracy in your paper.
The setting of batch_size=32 is found in previous issues.
![image](https://user-images.githubusercontent.com/26896476/128694539-bc49718f-cee2-46e8-9982-8362b26e7618.png)


 And I have tried to change the hyperparameters by myself, like this:
![image](https://user-images.githubusercontent.com/26896476/128694327-3b8fad9d-8d82-4c39-913c-8967c9203647.png)

**Comment by villawang** [comment]
Please refer to [linear scale rule](https://research.fb.com/wp-content/uploads/2017/06/imagenet1kin1h5.pdf).
